app/views/devise/registrations/new.html.erb

```diff
@@ -1,43 +1,45 @@
-<h2>Sign up</h2>
+<h1 class="h4 text-gray-900 mb-4">Create an Account!</h1>
 
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
-  <div class="field">
+  <div class="form-group">
     <%= f.label :name %><br />
-    <%= f.text_field :name%>
+    <%= f.text_field :name, class: "form-control",placeholder: "Enter Name Here", required: true %>
```

Comment:
Single quotes and internationalization. Verify everywhere.

Reply:
done